Packs/SysAid/Integrations/SysAid/SysAid.yml

```diff
@@ -111,682 +90,537 @@ script:
+      name: custom_fields_values
+    description: Create a new service record and return the newly created service record. The valid statuses can be retrieved using the "sysaid-table-list" command. Cortex XSOAR recommends filtering the results by the desired fields.
+    name: sysaid-service-record-create
     outputs:
-    - type: String
-      contextPath: SysAid.ServiceRecord.id
-      description: 'The ID of the service record.'
-    - type: String
-      contextPath: SysAid.ServiceRecord.title
-      description: 'The title of the service record.'
-    - type: String
-      contextPath: SysAid.ServiceRecord.status
-      description: 'The status of the service record.'
-    - type: Date
-      contextPath: SysAid.ServiceRecord.update_time
-      description: 'The modify time of the service record.'
-    - type: String
-      contextPath: SysAid.ServiceRecord.sr_type
-      description: 'The type of the service record.'
-    - type: String
-      contextPath: SysAid.ServiceRecord.notes
-      description: 'The notes of the service record.'
-  - name: sysaid-service-record-delete
-    description: 'Delete one or more service records.'
-    arguments:
-    - name: ids
-      description: 'The service record ID.'
+    - contextPath: SysAid.ServiceRecord.id
+      description: The ID of the service record.
+      type: String
+    - contextPath: SysAid.ServiceRecord.title
+      description: The title of the service record.
+      type: String
+    - contextPath: SysAid.ServiceRecord.status
+      description: The status of the service record.
+      type: String
+    - contextPath: SysAid.ServiceRecord.update_time
+      description: The modify time of the service record.
+      type: Date
+    - contextPath: SysAid.ServiceRecord.sr_type
+      description: The type of the service record.
+      type: String
+    - contextPath: SysAid.ServiceRecord.notes
+      description: The notes of the service record.
+      type: String
+  - arguments:
+    - description: The service record ID.
+      name: ids
+      required: true
+    - description: The solution for deleting the service record.
+      name: solution
+    description: Delete one or more service records.
+    name: sysaid-service-record-delete
+  - arguments:
+    - description: The service record ID.
+      name: id
+      required: true
+    - description: File file ID to upload.
+      name: file_id
+      required: true
+    description: Add an attachment to a service record.
+    name: sysaid-service-record-attach-file
+  - arguments:
+    - description: The service record ID.
+      name: id
+      required: true
+    - description: The attachment file ID to delete.
+      name: file_id
       required: true
-      isArray: false
-    - name: solution
-      description: 'The solution for deleting the service record.'
-      required: false
-      isArray: false
+    description: Delete an attachment from a service record.
+    name: sysaid-service-record-delete-file
```

Comment:
Please add the outputs path as well. You can use: [demisto-sdk generate-outputs](https://github.com/demisto/demisto-sdk/tree/master/demisto_sdk/commands/generate_outputs)